cirq-google/cirq_google/engine/engine_processor.py

```diff
@@ -110,8 +110,11 @@ def get_sampler(
         Returns:
             A `cirq.Sampler` instance (specifically a `engine_sampler.ProcessorSampler`
             that will send circuits to the Quantum Computing Service
-            when sampled.1
+            when sampled.
         """
+        if self._processor is not None:
```

Comment:
Should this rely on `_inner_processor()`? The `Engine*` classes lazy load their resources as needed, so it's possible to get an `EngineProcessor` object that doesn't have `self._processor` set although it could be loaded.

Reply:
Updated